code/frameworks/sveltekit/template/stories_svelte-kit-skeleton-js/links.stories.js

```diff
@@ -0,0 +1,26 @@
+import { expect, fn, within } from '@storybook/test';
+import Links from './Links.svelte';
+
+export default {
+  title: 'stories/sveltekit/modules/Links',
+  component: Links,
+  tags: ['autodocs'],
+};
+
+const link = fn();
+
+export const Link = {
+  async play({ canvasElement }) {
+    const canvas = within(canvasElement);
+    const button = canvas.getByText('Storybook');
+    button.click();
+    expect(link).toHaveBeenCalled();
```

Comment:
I guess an `expect` here about the location not to have been changed would be great too.